v3/setup/klam-ssh.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+AZ=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+REGION=${AZ::-1}
+ROLE_NAME="$(etcdctl get /klam-ssh/ROLE_NAME)"
+ENCRYPTION_ID="$(etcdctl get /klam-ssh/ENCRYPTION_ID)"
+ENCRYPTION_KEY="$(etcdctl get /klam-ssh/ENCRYPTION_KEY)"
+KEY_LOCATION_PREFIX="$(etcdctl get /klam-ssh/KEY_LOCATION_PREFIX)"
+IMAGE="$(etcdctl get /images/klam-ssh)"
+
+
+if [[ $REGION == "eu-west-1" ]]; then
+    KEY_LOCATION="-ew1"
+elif [[ $REGION == "ap-northeast-1" ]]; then
+    KEY_LOCATION="-an1"
+elif [[ $REGION == "us-east-1" ]]; then
+    KEY_LOCATION="-ue1"
+elif [[ $REGION == "us-west-1" ]]; then
+    KEY_LOCATION="-uw1"
+elif [[ $REGION == "us-west-2" ]]; then
+    KEY_LOCATION="-uw2"
+else
+    echo "An incorrect region value specified"
+    exit 1
+fi
+
+# create nsswitch.conf
```

Comment:
If "unhappy"; then
  how about a case statement?
elif "unsatisfied"; then
  how about a case statement?
else
  you get a case statement
  exit 0